firefox-ios/firefox-ios-tests/Tests/UnitTest.xctestplan

```diff
@@ -251,9 +197,7 @@
     },
     {
       "skippedTests" : [
-        "AnalyticsSpyTests\/testTrackMenuStatus()",
-        "AppDelegateFeatureManagementIntegrationTests\/testStateAfterDidBecomeActive_expectesSameModel_AfterDidFinishLaunchingWithOptions()",
-        "AppDelegateFeatureManagementIntegrationTests\/testStateAfterDidFinishLaunchingWithOptions_expectsModelUpdates()"
+        "AnalyticsSpyTests\/testTrackMenuStatus()"
```

Comment:
I guess this one is still flaky and will be harderd to fix? Noticed you made some improvements in it but I guess it's still flaky.

Reply:
Yeah, I wanted to make sure we would review it or rethink the test altogether. 
Skipping it for now as I realised it mostly fails locally (not CI) but still don't want this to hunt us back all of a sudden.